SampleProject/.gitignore

```diff
@@ -0,0 +1,4 @@
+/bin
```

Comment:
Hey we already have gitignore can you remove this?

Reply:
ok. I will remove this sorry.